.azure/pipelines/ci.yml

```
@@ -260,9 +257,10 @@ stages:
                 -buildInstallers
                 -noBuildNative
                 /p:DotNetSignType=$(_SignType)
-                /p:AssetManifestFileName=aspnetcore-win-arm64.xml
+                /p:AssetManifestFileName=aspnetcore-win-assets.xml
```

Not a big deal but I suggest

```suggestion
                /p:AssetManifestFileName=aspnetcore-win.xml
```

because the only other Windows asset manifest file is named `aspnetcore-win-arm.xml` and none of the 12 (soon to be 11) names include `assets`.